final_task/rss_reader/items.py

```diff
@@ -0,0 +1,81 @@
+import logging
+from dataclasses import dataclass
+from typing import List
+from feedparser import FeedParserDict
+from parser_rss import format_description
+
+
+@dataclass()
+class Item:
+    title: str
+    date: str
+    link: str
+    text: str
+    img_links: List[str]
+
+
+def print_items(items):
+    """ Print all items in stdout.
+
+    :param items: just all items
+    :type items: list of Item
+    """
+    for item in items:
+        print(f'\nTitle: {item.title}'
+              f'\nDate: {item.date}'
+              f'\nLink: {item.link}'
+              f'\nText: {item.text}')
+
+        if item.img_links:
+            print('Image links: ')
+            for num, link in enumerate(item.img_links):
+                print(f'\t[{num+1}]: [{link}]')
+
+
+def item_to_dict(item):
+    """ Convert Item to dict
+
+    :param item: just item
+    :type item: 'items.Item'
+
+    :raise TypeError: if type of item is not 'items.Item'
+
+    :return: converted item
+    :rtype: dict
+    """
+    if not isinstance(item, Item):
+        raise TypeError(f'Object with type {type(item)} is not serializable. Expected type: {Item}')
+
+    return {'title': item.title,
```

Comment:
`dataclasses` lib has function `as_dict` which you can use to cast your dataclass object into a dictionary

Reply:
fixed